js/ui/slider.d.ts

```diff
@@ -42,9 +43,6 @@ export type OptionChangedEvent = EventInfo<dxSlider> & ChangedOptionInfo;
 /** @public */
 export type ValueChangedEvent = NativeEventInfo<dxSlider, KeyboardEvent | MouseEvent | PointerEvent | TouchEvent | UIEvent | Event> & ValueChangedInfo;
 
-/** @public */
-export type ValueChangeMode = 'onHandleMove' | 'onHandleRelease';
-
```

Comment:
We can't remove public types that easy. Aliases required to keep backward compatibility (both in js/ui/slider.d.ts and in ts/dx.all.d.ts)

```ts
/**
 * @deprecated Use /common/SliderValueChangeMode instead
 */
export type ValueChangeMode = SliderValueChangeMode;
```

Reply:
done